ais-dashboard/account_interventions_service.json

```diff
@@ -0,0 +1,4058 @@
+{
+  "metadata": {
+    "configurationVersions": [
+      7
+    ],
+    "clusterVersion": "1.284.110.20240210-104704"
+  },
+  "id": "8730e683-d596-49d4-bd9c-2f66ee886b17",
+  "dashboardMetadata": {
+    "name": "[WIP] Accounts Intervention Service",
```

Comment:
Is this still a draft dashboard or can we remove `[WIP]`?

Reply:
Yeah I'm going to remove WIP, I think we should be happy with it 👍